kani_metadata/src/artifact.rs

```diff
@@ -0,0 +1,102 @@
+// Copyright Kani Contributors
+// SPDX-License-Identifier: Apache-2.0 OR MIT
+//! Represent information about an artifact type.
+
+use std::ffi::OsStr;
+use std::ops::Deref;
+use std::path::{Path, PathBuf};
+
+#[derive(Clone, Copy, Debug, Eq, Hash, PartialEq)]
+pub enum ArtifactType {
```

Comment:
Should have at least a short doc string. E.g. just saying this corresponds to a file extension and not something more meaningful.

Reply:
For sure. I need to document the enum and its variants. I'll do it in the next review.